### ISSUE TITLE
Customize Panes mode doesn't show scrollbar on central area

### ISSUE THREAD

**Issue by damienprystay** [Komodo/KomodoEdit#3082]
When customizing panes, if the window isn't tall enough to display all the toolbars in the central area, that area can be scrolled, but there's no visual indicator of it. Depending on the layout, this can mean the bottommost toolbars are hidden with no indication that they exist at all.

E.g. here Side Toolbar - Bottom is offscreen:
<img width="1213" alt="screen shot 2017-09-14 at 11 47 52" src="https://user-images.githubusercontent.com/3590102/30448091-a769c334-9942-11e7-892e-2cd7e06711fb.png">

Komodo IDE 11.0.0 build 90665
Mac OS X 10.11.6

**Comment by Naatan** [comment]
Can you reproduce this using the default color scheme?

**Comment by damienprystay** [comment]
Nope, I do see the scrollbar on Default, Classic, and a couple other light schemes. Looks like it's specifically the combination of greys in Londontube.

**Comment by Naatan** [comment]
Thanks @damienprystay !
